dclab/rtdc_dataset/plugins/plugin_feature.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. versionadded:: 0.34.0
+"""
+
+import pathlib
+import importlib
+import sys
+
+from ..ancillaries import AncillaryFeature
+
+
+_registered_plugin_instances = []
+
+
+def create_new_plugin_feature(plugin_path):
+    """find an instanciate a PlugInFeature from a user-defined script"""
+    info = find_plugin_feature_script(plugin_path)
+    plugin_list = []
+    for ii in range(len(info["feature names"])):
+        ref_info = {
+            "feature_name": info["feature names"][ii],
+            "method": info["method"],
+            "req_config": info["config required"][ii],
```

Comment:
Both `"req_config"` and `"req_features"` need to be list of lists for propper hashing, see `examples/plugin_example_features.py`. It isn't the cleanest look.